ProcessesApi.Tests/V1/Services/SoleToJointServiceTests.cs

```diff
@@ -850,13 +851,40 @@ public async Task ProcessStateIsUpdatedToScheduleTenureAppointmentOnHOApprovalPa
             // Assert
             CurrentStateShouldContainCorrectData(
                 process, trigger, SoleToJointStates.TenureAppointmentScheduled,
-                new List<string> { SoleToJointPermittedTriggers.CancelProcess });
+                new List<string> { SoleToJointPermittedTriggers.RescheduleTenureAppointment, SoleToJointPermittedTriggers.CancelProcess }
+             );
 
             process.PreviousStates.Last().State.Should().Be(SoleToJointStates.HOApprovalPassed);
             VerifyThatProcessUpdatedEventIsTriggered(SoleToJointStates.HOApprovalPassed, SoleToJointStates.TenureAppointmentScheduled);
         }
 
 
+        #endregion
+
+        #region Reschedule Tenure Appointment
+        [Fact]
+        public async Task ProcessStateIsUpdatedToRescheduleTenureAppointmentOnHOApprovalPassed()
```

Comment:
If you make this a theory with initialState as a param, you can check if you can do reschedule appointment from appointment rescheduled